src/handler.py

```diff
@@ -92,19 +98,19 @@ def get_legs(departure_airport, arrival_airport, departure_date):
         time.sleep(60)
         raise
 
-    time.sleep(10) # let the JS run
+    time.sleep(10)  # let the JS run
 
     request_time = dt.datetime.now()
     page = driver.page_source
     soup = BeautifulSoup(page, 'html.parser')
 
     result = []
-    for t in soup.find_all('li', {'class':'flight-module segment offer-listing'}):
+    for t in soup.find_all('li', {'class': 'flight-module segment offer-listing'}):
         # Duration
         raw_duration = t.find(lambda d: has_data_test_id("duration", d)).contents[0].strip()
         d = dt.datetime.strptime(raw_duration, "%Hh %Mm")
-        if (d.hour > 2): # don't save these long running ones
-            continue
+        if d.hour > 2:  # don't save these long running ones
+            continue  # TODO: this might cause problems if we start looking at long haul flights!
```

Comment:
Yeah but the code would need to change anyway if we did. This is to filter out those weird flights where it's like 'you can go from gatwick to amsterdam via Dubai and it'll only take 6 hrs'- I had trouble doing it based on the number of stops for some reasons. Remember this keeps flights up to 2h59m

Reply:
Makes sense. I will try to find time to pull it out to a configuration file or something, to avoid future pain.

Reply:
Meh don't think it's worth the trouble, this magic number is only in one place in the whole project

Reply:
OK